prow/cmd/checkconfig/main.go

```diff
@@ -793,6 +847,84 @@ func validateNeedsOkToTestLabel(cfg *config.Config) error {
 	return errorutil.NewAggregate(queryErrors...)
 }
 
+type filePresenceChecker interface {
+	remoteCheck(path string) (bool, error)
+	localCheck(path string, client *git.Client) bool
+}
+
+type _filePresenceChecker string
```

Comment:
nit: this is not common in Go. Identify what's special about this one in its name?

Reply:
You mean the underscore? I just needed something to carry the interface...